signac/common/config.py

```diff
@@ -99,45 +98,33 @@ def read_config_file(filename, configspec=None, *args, **kwargs):
     return config
 
 
-def load_config(root=None, local=False):
-    """Load configuration, searching upward from a root path if desired.
+def load_config(root=None):
```

Comment:
Does this file need to have _any_ public methods?

Reply:
That is a fair question. Ultimately this boils down to a question of how we expect users to interact with config files in signac 2.0. The files will be hidden, so most casual users probably won't think to do something like
```
import toml  # or whatever package
config = toml.open("/path/to/project/.signac/config") # or whatever the API looks like
```
We could really expect them to treat it like git and edit directly with a text editor, but that's also not really entry-level. I only muck with the git config manually if I really have to. That leaves the command-line config interface. I'm fine with that being the primary entry point as it currently is, the question is whether we're OK with that being the _only_ entry point. Is that what we want?

Reply:
I would be fine with making it CLI / manual editing only. We have never recommended using these APIs, all our docs and error/warning messages point to using the CLI.

Reply:
I recommend moving towards making all these functions private. Could be done in a later PR.

Reply:
Sounds good, I'll keep that on the docket for future work.